src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -240,6 +258,8 @@ public void resetGyroAngle(Rotation2 angle) {
         synchronized (sensorLock) {
             gyroscope.setAdjustmentAngle(
                     gyroscope.getUnadjustedAngle().rotateBy(angle.inverse()));
+            defaultX = gyroscope.getAxis(Pigeon.Axis.ROLL);
+            defaultY = gyroscope.getAxis(Pigeon.Axis.PITCH);
```

Comment:
?? This should be resetting based on the passed in `angle`.  This also seems to duplicate the line above, which will change what the gyroscope angle returns.